### PR TITLE
feat(Cypress): Add tests for the landing page and the navbar

### PR DESCRIPTION
**Pull request by Twaha-Rahman**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Related to #38611

This PR adds e2e tests for the `Navbar` in fCC's site. Let me know if anything needs to be changed!

### CODE REVIEW

**Comment by ahmadabdolsaheb** [comment]
@ojeytonwilliams should the sample.env FORUM_LOCATION be 'https://forum.localhost' instead of 'https://localhost/forum'

**Comment by ojeytonwilliams** [comment]
I'm not 100% sure. But, since we're using the subdomain in reality, https://forum.localhost makes sense to me.

**Comment by ahmadabdolsaheb** [comment]
Sure, we could leave it as is and address it in another pr where we would pull these values from a config file.

**Review by ahmaxed — DISMISSED** [review]

**Comment by raisedadead** [comment]
We should use the real URLs instead of localhost. The idea is to test if the link is valid and reachable.

**Comment by Twaha-Rahman** [comment]
> We should use the real URLs instead of localhost. The idea is to test if the link is valid and reachable.

That's a good idea cause that way we can see if the button is working correctly by clicking it and seeing if it takes us to the right site.

**Comment by Twaha-Rahman** [comment]
We'll need to update the environment variables to change those links I believe. Any thoughts?

**Comment by raisedadead** [comment]
Yes, and we should. I don't see any point keeping them to localhost.

**Comment by ahmadabdolsaheb** [comment]
Changing the # Application paths in sample.env should suffice.

**Comment by Twaha-Rahman** [comment]
Done! Updated the env variables and added a few more tests for the landing page.

File: cypress/integration/landing.js [hunk 1]
```diff
@@ -7,16 +7,40 @@ const selectors = {
...
+  });
+
+  it('Has 5 brand logos', () => {
+    cy.get('.logo-row')
+      .children()
+      .its('length')
+      .should('eq', 5);
+  });
+
+  it('Has testimonial section', () => {
```
Comment (ahmadabdolsaheb) [inline cypress/integration/landing.js:36]: It is not clear how it is testing the testimonials section.

Comment (Twaha-Rahman) [reply cypress/integration/landing.js]: Ooops! Sorry, my bad. The test name was really ambiguous. I've updated the test name now. Let me know what you think.

**Comment by ahmadabdolsaheb** [comment]
This LGTM. @ojeytonwilliams, it would be great if we could get this in so I could adjust the nav tests in the custom nav PR.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Comment by ojeytonwilliams** [comment]
Thanks for taking the time to put this together, @Twaha-Rahman, the extra code coverage is so important - this will be a massive help.

**Pull request merged**